src/app/gh-finder/modal.jsx

```diff
@@ -0,0 +1,93 @@
+import * as React from "react";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Modal from "@mui/material/Modal";
+import { Checkbox, FormControlLabel, FormGroup } from "@mui/material";
+import { Delete } from "@mui/icons-material";
+
+const style = {
+  position: "absolute",
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  width: 600,
+  height: 300,
+  border: "2px solid #000",
+  boxShadow: 24,
+  p: 4,
+  display: "flex",
+};
+
+export default function BasicModal({ selectedLabels, setSelectedLabels }) {
```

Comment:
Integrate these :

1. Search
2. Is assigned
3. Is fork

Reply:
Hi @Bashamega, I don't understand the use of "isFork is". Is it for displaying the forks in number or else?

Reply:
> Hi @Bashamega, I don't understand the use of "isFork is". Is it for displaying the forks in number or else?

I mean is there a fork opened for the issue. If someone have opened a fork for the issue

Reply:
Like 0 fork, 2k fork?

Reply:
> Like 0 fork, 2k fork?

No.

I mean if the issue has a fork to close it. So if for this repo if i checked it will hide issue #215 because you have opened a pr that will close it.

Reply:
Have a look 
{
    "id": 655318469,
    "node_id": "R_kgDOJw9dxQ",
    "name": "WebDevTools",
    "full_name": "Bashamega/WebDevTools",
    "private": false,
    "owner": {
        "login": "Bashamega",
        "id": 110662505,
        "node_id": "U_kgDOBpiTaQ",
        "avatar_url": "https://avatars.githubusercontent.com/u/110662505?v=4",
        "gravatar_id": "",
        "url": "https://api.github.com/users/Bashamega",
        "html_url": "https://github.com/Bashamega",
        "followers_url": "https://api.github.com/users/Bashamega/followers",
        "following_url": "https://api.github.com/users/Bashamega/following{/other_user}",
        "gists_url": "https://api.github.com/users/Bashamega/gists{/gist_id}",
        "starred_url": "https://api.github.com/users/Bashamega/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/Bashamega/subscriptions",
        "organizations_url": "https://api.github.com/users/Bashamega/orgs",
        "repos_url": "https://api.github.com/users/Bashamega/repos",
        "events_url": "https://api.github.com/users/Bashamega/events{/privacy}",
        "received_events_url": "https://api.github.com/users/Bashamega/received_events",
        "type": "User",
        "site_admin": false
    },
    "html_url": "https://github.com/Bashamega/WebDevTools",
    "description": "WebDev Tools is a comprehensive online platform designed to empower web developers with a wide array of code samples and snippets. ",
    "fork": false,
    "url": "https://api.github.com/repos/Bashamega/WebDevTools",
    "forks_url": "https://api.github.com/repos/Bashamega/WebDevTools/forks",
    "keys_url": "https://api.github.com/repos/Bashamega/WebDevTools/keys{/key_id}",
    "collaborators_url": "https://api.github.com/repos/Bashamega/WebDevTools/collaborators{/collaborator}",
    "teams_url": "https://api.github.com/repos/Bashamega/WebDevTools/teams",
    "hooks_url": "https://api.github.com/repos/Bashamega/WebDevTools/hooks",
    "issue_events_url": "https://api.github.com/repos/Bashamega/WebDevTools/issues/events{/number}",
    "events_url": "https://api.github.com/repos/Bashamega/WebDevTools/events",
    "assignees_url": "https://api.github.com/repos/Bashamega/WebDevTools/assignees{/user}",
    "branches_url": "https://api.github.com/repos/Bashamega/WebDevTools/branches{/branch}",
    "tags_url": "https://api.github.com/repos/Bashamega/WebDevTools/tags",
    "blobs_url": "https://api.github.com/repos/Bashamega/WebDevTools/git/blobs{/sha}",
    "git_tags_url": "https://api.github.com/repos/Bashamega/WebDevTools/git/tags{/sha}",
    "git_refs_url": "https://api.github.com/repos/Bashamega/WebDevTools/git/refs{/sha}",
    "trees_url": "https://api.github.com/repos/Bashamega/WebDevTools/git/trees{/sha}",
    "statuses_url": "https://api.github.com/repos/Bashamega/WebDevTools/statuses/{sha}",
    "languages_url": "https://api.github.com/repos/Bashamega/WebDevTools/languages",
    "stargazers_url": "https://api.github.com/repos/Bashamega/WebDevTools/stargazers",
    "contributors_url": "https://api.github.com/repos/Bashamega/WebDevTools/contributors",
    "subscribers_url": "https://api.github.com/repos/Bashamega/WebDevTools/subscribers",
    "subscription_url": "https://api.github.com/repos/Bashamega/WebDevTools/subscription",
    "commits_url": "https://api.github.com/repos/Bashamega/WebDevTools/commits{/sha}",
    "git_commits_url": "https://api.github.com/repos/Bashamega/WebDevTools/git/commits{/sha}",
    "comments_url": "https://api.github.com/repos/Bashamega/WebDevTools/comments{/number}",
    "issue_comment_url": "https://api.github.com/repos/Bashamega/WebDevTools/issues/comments{/number}",
    "contents_url": "https://api.github.com/repos/Bashamega/WebDevTools/contents/{+path}",
    "compare_url": "https://api.github.com/repos/Bashamega/WebDevTools/compare/{base}...{head}",
    "merges_url": "https://api.github.com/repos/Bashamega/WebDevTools/merges",
    "archive_url": "https://api.github.com/repos/Bashamega/WebDevTools/{archive_format}{/ref}",
    "downloads_url": "https://api.github.com/repos/Bashamega/WebDevTools/downloads",
    "issues_url": "https://api.github.com/repos/Bashamega/WebDevTools/issues{/number}",
    "pulls_url": "https://api.github.com/repos/Bashamega/WebDevTools/pulls{/number}",
    "milestones_url": "https://api.github.com/repos/Bashamega/WebDevTools/milestones{/number}",
    "notifications_url": "https://api.github.com/repos/Bashamega/WebDevTools/notifications{?since,all,participating}",
    "labels_url": "https://api.github.com/repos/Bashamega/WebDevTools/labels{/name}",
    "releases_url": "https://api.github.com/repos/Bashamega/WebDevTools/releases{/id}",
    "deployments_url": "https://api.github.com/repos/Bashamega/WebDevTools/deployments",
    "created_at": "2023-06-18T14:41:16Z",
    "updated_at": "2024-07-24T16:10:26Z",
    "pushed_at": "2024-07-23T18:33:04Z",
    "git_url": "git://github.com/Bashamega/WebDevTools.git",
    "ssh_url": "git@github.com:Bashamega/WebDevTools.git",
    "clone_url": "https://github.com/Bashamega/WebDevTools.git",
    "svn_url": "https://github.com/Bashamega/WebDevTools",
    "homepage": "https://wdt.adambashaahmednaji.com/",
    "size": 761,
    "stargazers_count": 30,
    "watchers_count": 30,
    "language": "JavaScript",
    "has_issues": true,
    "has_projects": false,
    "has_downloads": true,
    "has_wiki": false,
    "has_pages": false,
    "has_discussions": false,
    "forks_count": 27,
    "mirror_url": null,
    "archived": false,
    "disabled": false,
    "open_issues_count": 16,
    "license": {
        "key": "mit",
        "name": "MIT License",
        "spdx_id": "MIT",
        "url": "https://api.github.com/licenses/mit",
        "node_id": "MDc6TGljZW5zZTEz"
    },
    "allow_forking": true,
    "is_template": false,
    "web_commit_signoff_required": false,
    "topics": [
        "nextjs",
        "open-source",
        "tailwindcss",
        "webdev",
        "webdevtools"
    ],
    "visibility": "public",
    "forks": 27,
    "open_issues": 16,
    "watchers": 30,
    "default_branch": "main",
    "temp_clone_token": null,
    "network_count": 27,
    "subscribers_count": 2
}

Reply:
> Linked PR just like mine PR #215 on issue #158

yes. if he checked it all the issues that have a linked pr will disappear

Reply:
Hi @Adam-Basha, I have added this feature but this feature causing an Error 403 because of too many API calls.

`GET https://api.github.com/repos/Been777/fitple-project-backend 403 (Forbidden)`

Reply:
For getting **Linked PR** I have to add another fetch call and because of this GitHub API limit is exceeded.

Reply:
This function maps all the data and fetches linked PR
`  useEffect(() => {
    const fetchPRsInfo = async () => {
      const issuesWithPRsInfo = await Promise.all(
        data.map(async (issue) => {
          const linkedPRs = await fetchPRsForIssue(issue);
          return { ...issue, linkedPRs };
        }),
      );
      setData(issuesWithPRsInfo);
    };`